docs/INSTALLATION.md

```diff
@@ -0,0 +1,265 @@
+
+```sh
+pm2 start index.js -f -n "tuoris" -- -p PORT -i 1
+```
+
+To start the PM2 environment run:
+
+```sh
+OVE_HOST="OVE_CORE_HOST:PORT" TUORIS_HOST="TUORIS_HOST:PORT" pm2 start pm2.json
+```
+
+Once the services have started, you can check their status by running:
+
+```sh
+pm2 status
+```
+
+Then, to check logs of all services, run:
+
+```sh
+pm2 logs
+```
+
+To stop the PM2 environment run:
+
+```sh
+pm2 stop pm2.json
+```
+
+To clean-up the PM2 environment run:
+
+```sh
+pm2 delete pm2.json
+```
+
+### Compiling source code for the Docker runtime
+
+This approach only works for Linux and MacOS environments. The `build.sh` script corresponding to each repository can be found under the top most folder of the cloned or downloaded repository or within a `packages/PACKAGE_NAME` folder corresponding to each package.
+
+The `build.sh` script can be executed as:
+
+```sh
+cd ove
+./build.sh
+```
+
+Instructions above are only provided for the [**OVE Core**](https://github.com/ove/ove) repository. The steps to follow are similar for other repositories.
+
+#### Starting and stopping OVE
+
+Similar to the `build.sh` script, the `docker-compose.yml` file corresponding to each repository can also be found under the top most folder of the cloned or downloaded repository or within a `packages/PACKAGE_NAME` folder corresponding to each package.
+
+The deployment environment needs to be [pre-configured](#configuring-the-environment) before running these scripts.
+
+To start the Docker runtime run:
+
+```sh
+SERVICE_VERSION="latest" docker-compose -f docker-compose.yml up -d
+```
+
+Once the services have started, you can check their status by running:
+
+```sh
+docker ps
+```
+
+The `ps` command will list containers along with their `CONTAINER_ID`. Then, to check logs of an individual container, run:
+
+```sh
+docker logs <CONTAINER_ID>
+```
+
+To stop the Docker runtime run:
+
+```sh
+SERVICE_VERSION="latest" docker-compose -f docker-compose.yml down
+```
+
+To clean-up the Docker runtime first stop any active instances and then run:
+
+```sh
+docker system prune
+docker volume prune
+```
+
+## Running OVE
+
+It is recommended to use OVE with Google Chrome, as this is the web browser used for development and in production at the [Data Science Institute](http://www.imperial.ac.uk/data-science/). However, it should also be compatible with other modern web browsers: if you encounter any browser-specific bugs please [report them as an Issue](https://github.com/ove/ove-apps/issues).
+
+For details of how to use OVE, see the [Usage](./USAGE.md) page.
+
+After installation, OVE will expose several resources that can be accessed through a web browser:
+
+* App Control page   `http://OVE_APP_HOST:PORT/control.html?oveSectionId=0`
+* OVE client pages   `http://OVE_CORE_HOST:PORT/view.html?oveViewId=LocalNine-0` (check [`Spaces.json`](https://github.com/ove/ove/blob/master/packages/ove-core/src/client/Spaces.json) for more information)
```

Comment:
If you use a sub bullet point, this would make this easier to read
So:
```
* Point
 [tab] + Second point
```

Leads to

* OVE client pages   `http://OVE_CORE_HOST:PORT/view.html?oveViewId=LocalNine-0`
&ensp; + (check [`Spaces.json`](https://github.com/ove/ove/blob/master/packages/ove-core/src/client/Spaces.json) for more information)